### PR TITLE
Adds Syntax buildkite checks

### PR DESCRIPTION
**Pull request by afiune**
Signed-off-by: Salim Afiune <afiune@chef.io>

### CODE REVIEW

**Comment by afiune** [comment]
I don't know how to fix this error:
```
shellcheck: ./lib/scaffolding.sh: hGetContents: invalid argument (invalid byte sequence)
```
tenor-209080642

File: .expeditor/verify.pipeline.yml [hunk 1]
```diff
@@ -1,4 +1,13 @@
 steps:
+  - label: "Syntax"
+    command:
+      - apt-get update
+      - apt-get install -y shellcheck
+      - find . -name '*.sh' -print0 | xargs -0 -n1 shellcheck -e SC2034,SC2154 -s bash
+    plugins:
+      docker#v1.1.1:
+        image: ubuntu:16.04
```
Comment (srenatus) [inline .expeditor/verify.pipeline.yml]: The shellcheck page mentions that they publish docker images, too: grep for "docker" [here](https://github.com/koalaman/shellcheck#installing). I wonder if those images wouldn't save some time, by being smaller. I'm guessing the `koalaman/shellcheck` one won't work with the `find` call, but the `koalaman/shellcheck-alpine` might?

Comment (afiune) [reply .expeditor/verify.pipeline.yml]: Ill add this as an issue and merge this PR for now. We still want to start running shell checks now. 😄 

Thank you so much for keeping an eye on this @srenatus !! 💯

Comment (afiune) [reply .expeditor/verify.pipeline.yml]: https://github.com/chef/scaffolding-go/issues/6

**Pull request merged**